src/back/spv/image.rs

```diff
@@ -1121,7 +1104,7 @@ impl<'w> BlockContext<'w> {
                     self.get_type_id(
                         LocalType::Value {
                             vector_size: None,
-                            kind: crate::ScalarKind::Sint,
+                            kind: crate::ScalarKind::Uint,
                             width: 4,
                             pointer_space: None,
                         }
```

Comment:
We can just use `result_type_id` as we previously did.

https://github.com/gfx-rs/naga/blob/24316fd4bc9fca3b0c9cdd858ea593114c208eba/src/back/spv/image.rs#L1099-L1104

Reply:
thanks, didn't notice this